aliyun/src/test/java/org/apache/iceberg/aliyun/oss/AliyunTestUtility.java

```diff
@@ -25,17 +25,22 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-public class AliyunOSSTestUtility {
-  private static final Logger LOG = LoggerFactory.getLogger(AliyunOSSTestUtility.class);
-  private static final String ALIYUN_TEST_OSS_TEST_RULE_CLASS = "ALIYUN_TEST_OSS_TEST_RULE_CLASS";
+public class AliyunTestUtility {
```

Comment:
If this utility is not only for aliyun OSS, then I think it's better to move this class out of the oss package `org.apache.iceberg.aliyun.oss` ? maybe `org.apache.iceberg.aliyun` is a better choice ?